VVResonances/interactive/Projections3DHisto.C

```diff
@@ -107,6 +110,10 @@ for(int i=0; i<4; ++i){
  pullsy[i]->SetMarkerSize(0);
   
 }
+hx[0]->SetMinimum(0);
```

Comment:
The minimum are already set here

https://github.com/daniSchaefer/cmgtools-lite/blob/f0c4a6995cdfbfa675b744a7c2fc1ac639ffaf9c/VVResonances/interactive/Projections3DHisto.C#L127
https://github.com/daniSchaefer/cmgtools-lite/blob/f0c4a6995cdfbfa675b744a7c2fc1ac639ffaf9c/VVResonances/interactive/Projections3DHisto.C#L135

If you like you can add the maximum there too.